CMakeLists.txt

```diff
@@ -9,6 +9,8 @@ project(popt
 	LANGUAGES C
 )
 
+option(BUILD_SHARED_LIBS "Build using shared libraries" ON)
```

Comment:
This is already built into CMake, I think, so we don't need this here.

Reply:
Maybe it would be better to add a separate target to optionally produce a static library then?